.github/workflows/streamline-approval-workflow.yml

```diff
@@ -0,0 +1,55 @@
+name: Streamline-Approve
+
+on:
+  pull_request:
+    branches: ["master", "main"]
+
+jobs:
+  run-streamlined-approval:
+    if: github.event.pull_request.user.login == 'hs-pr-bot'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get Labels
+      id: getLabels
+      run: |
+        labels="$(gh api repos/$OWNER/$REPO_NAME/pulls/$PULL_REQUEST_NUMBER --jq '.labels.[].name')"
+        echo "allLabels=$(echo $labels)" >> $GITHUB_OUTPUT
+      shell: bash
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        OWNER: ${{ github.repository_owner }}
+        REPO_NAME: ${{ github.event.repository.name }}
+        PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+
+    - name: Checkout
+      if: "${{ contains(steps.getLabels.outputs.allLabels, 'auto-approve') }}"
+      uses: actions/checkout@v2.3.4
```

Comment:
@clpendleton Why are we using this 4yo release, rather than v4? Also, what are we doing with the checkout? Is it used by the steps that follow?

Reply:
@dduugg Thank you for bringing this to our attention! Checkout is actually not needed so I removed it.

Reply:
Woo 🎉 ! My favorite resolution method.